src/frontend/typer/registerytelser.ts

```diff
@@ -31,7 +31,7 @@ export const typeRegisterYtelseTilMålgruppeType: Record<TypeRegisterYtelse, Må
 };
 
 export const registerYtelseTilTekst: Record<TypeRegisterYtelse, string> = {
-    AAP: 'Arbeidsavklaringspenger',
-    ENSLIG_FORSØRGER: 'Overgangsstønad',
-    OMSTILLINGSSTØNAD: 'Omstillingsstønad',
+    AAP: 'arbeidsavklaringspenger',
+    ENSLIG_FORSØRGER: 'overgangsstønad',
+    OMSTILLINGSSTØNAD: 'omstillingsstønad',
```

Comment:
Disse brukes fra andre steder og, der det nog er ønskelig med capital letter 👀

Reply:
Litt usikker på om det er ønskelig å vise "AAP" i stedet for `arbeidsavklaringspenger` på akkurat denne siden

Reply:
Vi vet det, men ytelser i NAV skrives med liten bokstav. Se f.eks. https://arc.net/l/quote/hvzhaqwx

Unntaket er selvsagt hvis det står på starten av en setning, men det er det ikke denne funksjonen sin oppgave å passe på.

Reply:
Jeg gjorde det enkleste jeg kunne komme på som var å lage en egen oppslagstabell med stor forbokstav. Det virker unødvendig å skulle modde på strenger når de er statiske og vi vet hva de skal være.